daemon/daemon_unix.go

```diff
@@ -54,9 +54,16 @@ import (
 const (
 	isWindows = false
 
+	// These values were used to adjust the CPU-shares for older API versions,
+	// but were not used for validation.
+	//
+	// TODO(thaJeztah): validate min/max values for CPU-shares, similar to Windows:
+	// https://github.com/moby/moby/blob/27e85c7b6885c2d21ae90791136d9aba78b83d01/daemon/daemon_windows.go#L97-L99
+	//
 	// See https://git.kernel.org/cgit/linux/kernel/git/tip/tip.git/tree/kernel/sched/sched.h?id=8cd9234c64c584432f6992fe944ca9e46ca8ea76#n269
-	linuxMinCPUShares = 2
-	linuxMaxCPUShares = 262144
+	// linuxMinCPUShares = 2
+	// linuxMaxCPUShares = 262144
+
```

Comment:
Oh! I should probably create a tracking ticket for this (at least to discuss if we need validation on our side)

Reply:
created a tracking ticket: I'll update this commit to refer to that ticket (for easier discovery)

- https://github.com/moby/moby/issues/47340